domain/src/main/java/com/example/domain/Calculator.kt

```diff
@@ -0,0 +1,59 @@
+package com.example.domain
+
+class Calculator {
+
+    private var opertaion: Operation? = null
+    private var result = 0f
+
+    fun evaluate(inputs: String): Float {
+        initValue()
+        inputs.forEachIndexed { index, _value ->
+            val value = _value.toString()
+            val type = index % 2
+            when (type) {
```

Comment:
```suggestion
            when (index % 2) {
```
type 변수는 다른 곳에서 활용하는 곳이 없기 때문에 불필요해보여요

Reply:
수정하였습니다.